src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+abstract class Machine {
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+}

```

Comment:
The `Machine` class should be public to ensure it can be accessed from other packages if needed, which is typically the case for an abstract class that will be subclassed in different packages.